app/models/wise_transfer.rb

```diff
@@ -207,4 +210,20 @@ def generate_quote!
     update!(quoted_usd_amount_cents: estimated_usd_amount_cents) unless quoted_usd_amount_cents.present?
   end
 
+  WISE_ID_REGEXP = /wise.com\/transactions\/activities\/by-resource\/transfer\/(\d+)/i
+
+  def normalize_wise_id
```

Comment:
Aren't we doing this on the front end?

Reply:
Ah yeah we are https://github.com/hackclub/hcb/blob/3fd965acd69eeb4d3268f2f1a14cfcedebf27e24/app/views/admin/wise_transfer_process.html.erb#L180-L186

but also, never trust the front-end alone